ConfigVersionControl/git_version_control.py

```diff
@@ -126,11 +126,15 @@ def add(self, path):
         Args:
             path (str): the file to add
         """
+        if self._should_ignore(path):
+            return
+        if not self._needs_adding(path):
+            print "GIT: unchanged or already added '{}'".format(path)
+            return # unchanged or already added
+        print "GIT: adding '{}' ".format(path)
```

Comment:
Blockserver has a `print_and_log` method which should be used as this also writes to a log file which is much more useful.